ci/sdk_go.go

```diff
@@ -7,26 +7,21 @@ import (
 	"strings"
 
 	"github.com/moby/buildkit/identity"
-	"golang.org/x/sync/errgroup"
 
 	"github.com/dagger/dagger/ci/consts"
-	"github.com/dagger/dagger/ci/util"
 )
 
 type GoSDK struct {
 	Dagger *Dagger // +private
 }
 
-// Lint the Go SDK
+// Lint the Go SDK source code
 func (t GoSDK) Lint(ctx context.Context) error {
-	eg, ctx := errgroup.WithContext(ctx)
-	eg.Go(func() error {
-		return t.Dagger.Go().Lint(ctx, []string{"sdk/go"}, false)
```

Comment:
Okay, so we're not using this module for go linting anymore.

What's the end game for this? Is the idea to do all the linting in each module for each SDK? Which is a bit less trivial because of the interfaces? (but should still be possible today).

If that's the case, should we remove the linting functionality from the `ci/std/go` module for the time-being? I'd rather not have duplicate code, one of which isn't used (makes grepping through code, etc, much more confusing).